package-lock.json

```diff
@@ -4,6 +4,15 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@angular/common": {
```

Comment:
Sorry, I was unclear in my previous comment. 

You included `package-lock.json` which is great! But we will also need to include `@angular/common` in our `package.json`, under `devDependencies`, otherwise will `npm` remove these lines from `package-lock.json` next time we run `npm install` and we can no longer build the project. 
We should also add `@angular/common` to `peerDependencies` to make sure the user of this lib have `@angular/common` installed.

So lets
1. Add `@angular/common` to `package.json`, by running `npm install @angular/common --save-dev`
2. Add `@angular/common` to `peerDependencies`:
```diff
 "peerDependencies": {
   "@angular/core": ">=7.0.0",
+  "@angular/common": ">=7.0.0",
   "rxjs": ">=6.0.0"
 },
```
3. Add and commit both `package.json` and `package-lock.json`

Thanks :)

Reply:
Perfect @tjoskar, I understand now, I going to send it now 😄